buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijVersions.kt

```diff
@@ -0,0 +1,33 @@
+package com.virtuslab.gitmachete.buildsrc
+
+// See https://www.jetbrains.com/intellij-repository/releases/ -> Ctrl+F .idea
+object IntellijVersions {
+
+    val intellijVersionsProp = IntellijVersionHelper.getProperties()
```

Comment:
`private`?

Reply:
Wasn't there a problem with private vals in Kotlin? Or was that something else?

Reply:
Huh there is, I forbade them with a regex xDD although this doesn't matter as much in build code, you can probably exclude them from `scripts/prohibit-kotlin-private-fields`